BranchMetrics/ios-branch-deep-linking-attribution
# [SDK-2055] Remove v1/event code

## Reference
SDK-2055 -- Remove v1/event code

## Summary
Removed existing code for deprecated, no-op v1/events (sendCommerceEvent and userCompletedAction). This required deleting the files and all references of `BNCCommerceEvent` and `BranchUserCompletedActionRequest`. Since `BNCProductCategory` and `BNCCurrency` were located in `BNCCommerceEvent`, they were moved to their own new respective files. 

## Motivation
Remove old code that has been deprecated for a sufficient amount of time.

## Type Of Change


## Testing Instructions

Integrate the SDK, run the app, and run the tests to ensure everything is still functioning correctly.









cc @BranchMetrics/saas-sdk-devs for visibility.
